src/commands/EventCommand.js

```diff
@@ -70,66 +70,73 @@ class EventCommand extends SubsCommand {
         const channel = message.channel;
         const types = ["event", "task"];
 
-        const builder = new CommandBuilder("event.create", message.author, channel, [
-            {
+        const builder = new CommandBuilder("event.create", message.author, channel, [{
                 "name": "type",
                 "example": types,
                 "validate": (content) => {
-                    if (!types.includes(content)) 
+                    if (!types.includes(content))
                         return ["command.event.type-not-valid", types.join(", ")];
                     else
                         return true;
                 }
             },
             {
                 "name": "name",
```

Comment:
Can we please movetitle before name? Can we please add option "-" to name to auto-generate the name from the title (replacing spaces to -, making all chars lowercase and replacing non-basic chars (ř -> r))? Example of using aut-generated options, please look into this same file and lin 126 and lower.